modules/upsell-order-bump/includes/Blocks/OrderBumpCheckoutIntegration.php

```
@@ -0,0 +1,157 @@
+			$regular_price = $_product->get_regular_price();
+			// Use sale price if available, otherwise use regular price for discount calculation
+			$current_price = $_product->get_sale_price() ? $_product->get_sale_price() : $regular_price;
+			if ( 'discount' === $offer_type ) {
+				$offer_price = ( $current_price - ( $current_price * $offer_amount / 100 ) );
+			} else {
+				$offer_price = $offer_amount;
+			}
+
+			$cart                            = WC()->cart;
+			$product_already_added_from_shop = false;
+			foreach ( $cart->get_cart() as $cart_item_key => $cart_item ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$cart_item_key'. (undefined)

(UnusedLocalVariable)